nsxt/resource_nsxt_lb_pool.go

```diff
@@ -251,7 +251,7 @@ func setSnatTranslationInSchema(d *schema.ResourceData, snatTranslation *loadbal
 	if snatTranslation != nil {
 		if snatTranslation.Type_ == "LbSnatIpPool" {
 			elem["type"] = "SNAT_IP_POOL"
-			if snatTranslation.IpAddresses != nil && len(snatTranslation.IpAddresses) > 0 {
+			if len(snatTranslation.IpAddresses) > 0 {
```

Comment:
I did not know that nil slices have 0 len and you can check it without triggering a panic... nice.
I probably spent too many years working with Python.